artiq/frontend/artiq_dashboard.py

```diff
@@ -93,6 +89,12 @@ def main():
     args = get_argparser().parse_args()
     widget_log_handler = log.init_log(args, "dashboard")
 
+    if args.db_file is None:
+        args.db_file = os.path.join(get_user_config_dir(),
+                           "artiq_dashboard_{server}_{port}.pyon".format(
+                            server=args.server.replace(":","."),
```

Comment:
Is ``:`` actually forbidden in filenames on any platform? FWIW Linux uses it in sysfs.

Reply:
Its forbidden on Windows, alas.